api/pkg/transformer/pipeline/table_join_op.go

```diff
@@ -19,6 +20,9 @@ func NewTableJoinOp(tableJoinSpec *spec.TableJoin) Op {
 }
 
 func (t TableJoinOp) Execute(context context.Context, environment *Environment) error {
+	span, _ := opentracing.StartSpanFromContext(context, "pipeline.TableJoin")
```

Comment:
We can add more data to span by using SetTag:

```
span.SetTag("table.left", t.tableJoinSpec.LeftTable)
span.SetTag("table.right", t.tableJoinSpec.RightTable)
span.SetTag("table.output", t.tableJoinSpec.OutputTable)
```

Having this can help us identify the operation on an individual level, especially when there are multiple similar operations.